install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -699,7 +699,7 @@ spec:
       {{- if and .Values.ipMasqAgent .Values.ipMasqAgent.enabled }}
       - name: ip-masq-agent
         configMap:
-          name: ip-masq-agent
+          name: cilium-config
```

Comment:
I find this a bit odd - the idea here today is that Cilium can take whatever existing ip-masq-agent config you have (used even with non-Cilium ipmasq-agent), and read, understand and implement it. Is that not the case today?

Reply:
Thanks! 

Do you mean we need to create a `configmap` with the name "ip-masq-agent" here, and then we refer to it here?

Reply:
Yes, that's my understanding. This line should not need to change at all. As far as I know, you just need to create the ip-masq-agent configuration configmap in the cluster.

Reply:
Agree! Thanks.